src/lib/dev-environment/dev-environment-cli.ts

```diff
@@ -410,12 +411,22 @@ async function processComponent(
 	);
 	let result: ComponentConfig;
 
-	const allowLocal = true;
+	let allowLocal: boolean = true;
+
+	if ( component === 'muPlugins' ) {
+		try {
+			const currentUser = await getCurrentUserInfo( true );
+			allowLocal = currentUser?.isVIP ?? false;
+		} catch ( err ) {
+			allowLocal = false;
+		}
+	}
+
 	const defaultObject = defaultValue
-		? processComponentOptionInput( defaultValue, allowLocal )
+		? processComponentOptionInput( defaultValue, allowLocal as unknown as true )
```

Comment:
This is ugly, but there is no other sane option except rewriting `processComponentOptionInput`.

Reply:
😢